app/views/articles/_form.html.erb

```diff
@@ -32,14 +32,15 @@
 
         <div class="label-select-article">
           <%= form.label :status %><br>
-          <%= form.select :status, ['public', 'private', 'archived'], selected: 'public', id: "select-article" %>
+          <%= form.select :status, ['public', 'private', 'archived'], selected: 'public' %>
         </div>
 
         <div class="label-submit-article">
+          <% form.select :owner, [current_user.id] %>
```

Comment:
Why is it sent with a select? It should be sent with a hidden field

Reply:
It´s true, sorry, it was a provisional option since it did not take into account all the options of the form, I will make this change for the next pull request, thanks for the advise

Reply:
Like the previous comments, these types of changes are made in the same PR since they are part of the same feature